bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-streamingfile/bitsail-connector-streamingfile-core/src/main/java/com/bytedance/bitsail/connector/legacy/streamingfile/sink/FileSystemSinkFunctionDAGBuilder.java

```diff
@@ -117,6 +117,7 @@ private void addSinkConf(BitSailConfiguration jobConf) {
       return;
     }
     if (HIVE_FORMAT_TYPE_VALUE.equalsIgnoreCase(formatType)) {
+      //TODO: whether to load from hive conf location?
       Preconditions.checkState(jobConf.fieldExists(FileSystemSinkOptions.HIVE_METASTORE_PROPERTIES));
```

Comment:
@hk-lrzy not quite sure about this check, is this config a must have option here?